### PR TITLE
added comments and organised the metadata according to cuds components

### PR DESCRIPTION
**Pull request by ahashibon**
@mehdisadeghi please check this simple pull request, it is meant to ease searching for the cuds components in the file.

### CODE REVIEW

File: yaml_files/simphony_metadata.yml [hunk 1]
```diff
@@ -102,49 +115,14 @@ CUDS_KEYS:
     definition: Classical atomistic molecular dynamics using Newtons equations of motion
     parent: CUBA.PHYSICS_EQUATION
     models: [CUBA.ATOMISTIC]
-    variables: [CUBA.POSITION, CUBA.VELOCITY, CUBA.MOMENTUM, CUBA.ACCELERATION, CUBA.FORCE]
+    variables: [CUBA.POSITION, CUBA.VELOCITY, CUBA.MOMENETUM, CUBA.ACCELERATION, CUBA.FORCE]
```
Comment (stefanoborini) [inline yaml_files/simphony_metadata.yml:118]: MOMENTUM? keyboard slip?

Comment (ahashibon) [reply yaml_files/simphony_metadata.yml:118]: yes!

**Comment by mehdisadeghi** [comment]
@ahashibon except the typo mentioned above looks good to me.

File: yaml_files/simphony_metadata.yml (hunk 1 above)
Comment (mehdisadeghi) [reply yaml_files/simphony_metadata.yml:118]: @ahashibon CUBA.MOMENETUM or CUBA.MOMENTUM?

**Pull request merged**